tutorials/large/L2_large_link_prediction.py

```diff
@@ -356,6 +356,106 @@ def closure():
                 break
 
 
+######################################################################
+# Evaluating Performance with Link Prediction (Optional)
+# ------------------------------------------------------
+#
+# In practice, it is more common to evaluate the link prediction
+# model to see whether it can predict new edges.  There are different
+# evaluation metrics such as
+# `AUC <https://en.wikipedia.org/wiki/Receiver_operating_characteristic#Area_under_the_curve>`__
+# or `various metrics from information retrieval <https://en.wikipedia.org/wiki/Evaluation_measures_(information_retrieval)>`__.
+# Ultimately, they require the model to predict one scalar score given
+# a node pair among a set of node pairs.
+#
+# ``dgl.dataloading.EdgeDataLoader`` allows you to iterate over
+# the edges of a new graph with the same nodes, while performing
+# neighbor sampling on the original graph with ``g_sampling`` argument.
+# This functionality enables convenient evaluation of a link prediction
+# model.
+#
+# Assuming that you have the following test set with labels, where
+# ``test_pos_src`` and ``test_pos_dst`` are ground truth node pairs
+# with edges in between (or *positive* pairs), and ``test_neg_src``
+# and ``test_neg_dst`` are ground truth node pairs without edges
+# in between (or *negative* pairs).
+#
+
+# Positive pairs
+test_pos_src, test_pos_dst = graph.edges()
+# Negative pairs
+test_neg_src = test_pos_src
+test_neg_dst = torch.randint(0, graph.num_nodes(), (graph.num_edges(),))
+
+
+######################################################################
+# First you need to construct a graph for ``dgl.dataloading.EdgeDataLoader``
+# to iterate on, which should have the same nodes as the original
+# ``graph``, but the testing node pairs as edges.  You also need to
+# label the edges with 1 if the corresponding node pair is positive,
+# and 0 if negative:
```

Comment:
"label the edges with 1 if the corresponding node pair is positive, and 0 if negative" -> "label the edges, 1 for positive and 0 for negative"